src/system/UsersModule/Entity/Repository/UserRepository.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Copyright Zikula Foundation 2015 - Zikula Application Framework
+ *
+ * This work is contributed to the Zikula Foundation under one or more
+ * Contributor Agreements and licensed to You under the following license:
+ *
+ * @license GNU/LGPLv3 (or at your option, any later version).
+ * @package Zikula_Form
+ *
+ * Please see the NOTICE file distributed with this source code for further
+ * information regarding copyright and licensing.
+ */
+
+namespace Zikula\UsersModule\Entity\Repository;
+
+use Doctrine\ORM\EntityRepository;
+use Zikula\UsersModule\Entity\UserEntity;
+use Zikula\UsersModule\Entity\RepositoryInterface\UserRepositoryInterface;
+
+class UserRepository extends EntityRepository implements UserRepositoryInterface
+{
+    public function findByUids($uids)
```

Comment:
I'm not sure I understand what you mean. Do you mean simply renaming the method? To what would you suggest? I'm not sure I would agree, because no matter the repository, the method indeed is *find*ing users by a `uid` array.